.travis.yml

```diff
@@ -22,10 +25,24 @@ jobs:
       name: "OpenJDK 8"
       jdk: openjdk8
       script: mvn clean verify -B
+    - stage: test
```

Comment:
Can we enable this only on master?

Reply:
Why stage:test here too?

Reply:
I have enable the openjdk tests only for master branch and removed any reductant tags (such as stage)